web/src/components/merchant/sign-in/hooks/useSignInForm.tsx

```diff
@@ -35,20 +38,36 @@ const useSignInForm = () => {
     mode: 'onChange',
   });
   const [generalError, setGeneralError] = useState();
+  const history = useHistory();
 
   const validations = {
     email: register({
-      required: Errors.EMAIL_EMPTY,
+      required: true,
     }),
     password: register({
-      required: Errors.PASSWORD_EMPTY,
+      required: true,
     }),
   };
   const disabled = !formState.isValid;
 
   const onSubmit = handleSubmit(async (values: SignInData) => {
     setGeneralError(null);
```

Comment:
Actually I don't think your `generalError` takes in a null type.

Reply:
Oh that's weird I didn't have any error :/